addons/binding/org.openhab.binding.nest.test/META-INF/MANIFEST.MF

```diff
@@ -1,26 +1,50 @@
 Manifest-Version: 1.0
+Bundle-ActivationPolicy: lazy
 Bundle-ManifestVersion: 2
-Bundle-Name:  Nest Binding Tests
+Bundle-Name: Nest Binding Tests
 Bundle-RequiredExecutionEnvironment: JavaSE-1.8
 Bundle-SymbolicName: org.openhab.binding.nest.test;singleton:=true
 Bundle-Vendor: openHAB
 Bundle-Version: 2.3.0.qualifier
-Export-Package: org.openhab.binding.nest;uses:="org.eclipse.smarthome.te
- st"
+Export-Package: 
+ org.openhab.binding.nest;uses:="org.eclipse
+ .smarthome.test",
+ org.openhab.binding.nest.handler;uses:="org.eclipse
+ .smarthome.test",
+ org.openhab.binding.nest.test;uses:="org.eclipse
+ .smarthome.test"
 Fragment-Host: org.openhab.binding.nest
 Import-Package: 
+ com.google.gson.reflect;version="2.7.0",
+ javax.measure,
+ javax.measure.quantity,
+ javax.servlet,
+ javax.servlet.http,
  org.eclipse.jdt.annotation;resolution:=optional,
+ org.eclipse.jetty.server;version="[9.3.15,9.4.0)",
+ org.eclipse.jetty.server.handler;version="[9.3.15,9.4.0)",

```

Comment:
See above

Reply:
It fails to resolve the correct bundles when I remove these versions.